webaccess/src/webaccess_ja_JP.ts

```diff
@@ -0,0 +1,345 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="fr_FR">
+<context>
+    <name>WebAccess</name>
+    <message>
+        <location filename="webaccess.cpp" line="169"/>
+        <source>Loading project...</source>
+        <translation>プロジェクトを開いています...</translation>
```

Comment:
This and other strings ending with . are warning for Qt Linguist.
However, I have no idea if in Japanese you use punctuation at the end of sentences.
Up to you.

Reply:
No problem.
Three dot mean please wait in japanese.
It is not correct grammar. But we are useing it in everyday.